src/components/User/Limits/components/Limits.Manager.vue

```diff
@@ -1,237 +1,313 @@
+                class="limits-manager__update-btn app__btn app__btn--info"
+                :disabled="isPending"
+                @click="updateLimits(limits.deposit)"
+              >
+                Update
+              </button>
+
+              <button
+                class="
+                limits-manager__remove-btn
+                app__btn app__btn-outline
+                app__btn-outline--danger
+              "
+                :disabled="isPending || limits.deposit.id === 0"
+                @click="removeLimits(limits.deposit, 'deposit')"
+              >
+                Remove
+              </button>
             </div>
-          </div>
-
-          <div class="limits-manager__limits-action">
-            <button
-              class="limits-manager__update-btn app__btn"
-              :disabled="isPending"
-              @click="updateLimits(limits.deposit)"
-            >
-              Update deposit limits
-            </button>
-          </div>
-        </template>
-      </div>
+          </template>
+        </div>
+      </template>
+      <template v-else-if="isAddressLoading">
+        <div class="limits-manager__message-wrp">
+          <p>Loading ...</p>
+        </div>
+      </template>
+      <template v-else-if="!specificUserAddress">
+        <div class="limits-manager__message-wrp">
+          <p>Please specify account</p>
+        </div>
+      </template>
+      <template v-else>
+        <div class="limits-manager__message-wrp">
+          <p>Such account not found</p>
+        </div>
+      </template>
     </div>
   </div>
 </template>
 
 <script>
 import { SelectField, InputField } from '@comcom/fields'
-import { Tabs, Tab } from '@comcom/Tabs'
 import get from 'lodash/get'
 import throttle from 'lodash/throttle'
 import pick from 'lodash/pick'
 import api from '@/api'
 import { Sdk } from '@/sdk'
-import { STATS_OPERATION_TYPES, DEFAULT_MAX_AMOUNT } from '@/constants'
+import {
+  STATS_OPERATION_TYPES,
+  DEFAULT_MAX_AMOUNT,
+  DEFAULT_INPUT_STEP,
+} from '@/constants'
 import config from '@/config'
+import { confirmAction } from '@/js/modals/confirmation_message'
 import { ErrorHandler } from '@/utils/ErrorHandler'
+import { ApiCallerFactory } from '@/api-caller-factory'
+import { LimitsRecord } from '@/js/records/limits.record'
+
 const LIMITS_TYPES = [
   'dailyOut',
   'weeklyOut',
   'monthlyOut',
   'annualOut',
 ]
-const TAB_NAMES = {
-  accountType: 'Specific account type',
-  account: 'Specific account',
-}
+
+const SCOPE_TYPES = Object.freeze({ // non-xdr values, internal use only
+  account: 'USER',
+  accountRole: 'ACCOUNT_TYPE',
+  global: 'GLOBAL',
+})
+
 export default {
   components: {
     SelectField,
     InputField,
-    Tabs,
-    Tab,
   },
   data: _ => ({
     filters: {
       asset: '',
       accountRole: '',
-      requestor: '',
+      address: '',
+      scope: SCOPE_TYPES.global,
     },
-    selectedTabName: '',
+    specificUserAddress: '',
+    userEmail: '',
```

Comment:
1. `specificUserAddress` and related logic should be included in `filters` and renamed into `account`
2. `filters.address` should be removed completely
3. `userEmail` should be removed completely. use new `filters.account` instead
4. a new `userAddress` should be introduced, it is going to be treated like the current `filters.address`

the problem is that you got too many variables that are derivatives of the address entered by the user, and the address itself positioned not in the `filters` object, that actually contains all the variables entered by the user, but near of it, also the `filters.address` was actually filled out by the component. confuses a lot